src/pages/FAQ.tsx

```diff
@@ -54,30 +60,74 @@ export default function FAQ() {
         </div>
 
         <div className="flex flex-col items-center max-w-3xl px-4">
-          <ul className="list-none py-2 space-y-12">
+          <div className="py-2 space-y-3">
             {searchResults.map((FAQ, i) => {
               const { question, answer } = FAQ;
               return (
-                <li className="space-y-4" key={`q-${i}`}>
-                  <div className="font-bold text-lg">
-                    <strong>{question}</strong>
+                <div key={`q-${i}`} className="">
```

Comment:
`key={i}` works too. And the className is empty.
```suggestion
                <div key={i}>
```